recipe/meta.yaml

```
@@ -8,6 +8,8 @@ package:
 source:
   url: https://github.com/numpy/numpy/releases/download/v{{ version }}/numpy-{{ version }}.tar.gz
   sha256: c93733dbebc2599d2747ceac4b18825a73767d289176ed8e02090325656d69aa
+  patches:
+    - macos-lto-check.patch
 
 build:
   number: 0
```

As we added a patch, we should probably bump the build number.

```suggestion
   number: 1
```

---

Isn't that patch only for 3.8?

---

Seems to be applied to all Python version builds.

---

It's only needed for 3.8 but only because the others don't have `-flto` in the compiler flags. I see no harm in applying it to all builds given it has no effect on the build products.

---

> It's only needed for 3.8 but only because the others don't have `-flto` in the compiler flags. I see no harm in applying it to all builds given it has no effect on the build products.

Actually, that is good to have so we know our patches are upstreamable.

---

Indeed, I was just about make a PR to numpy 👍